doc/lsst.ap.pipe/pipeline-tutorial-gen2.rst

```diff
@@ -0,0 +1,265 @@
+   In the examples above, it is configured with the ``-c`` option, but a personal config file may be more convenient if you intend to run ``ap_pipe`` many times.
+
+.. _section-ap-pipe-expected-outputs-gen2:
+
+Expected outputs
+================
+
+If you used the rerun option above, most of the output from ``ap_pipe`` should be written out in the repo/rerun/processed directory,.
+The exception is the source association database, which will be written to the location you configure.
+The result from running ``ap_pipe`` should look something like
+
+.. code-block:: none
+
+   apdb/
+      association.db   <--- the Alert Production Database with DIAObjects
+   repo/
+      rerun/
+         processed/
+            repositoryCfg.yaml
+            deepDiff/
+               v123456/   <--- difference images and DIASource tables are in here
+            123456/   <--- all other processed data products are in here (calexps etc.)
+
+This is one example, and your rerun or output directory structure may differ.
+Of course, to inspect this data with the Butler, you don't need to know
+where it lives on disk. You should instead instantiate a Butler within python
+in the ``processed`` directory and access the data products that way.
+
+For example, in python
+
+.. code-block:: python
+
+   import lsst.daf.persistence as dafPersist
+   butler = dafPersist.Butler('repo/rerun/processed')
+   dataId = {'visit': 123456, 'ccdnum': 42, 'filter': 'g'}
+   calexp = butler.get('calexp', dataId=dataId)
+   diffim = butler.get('deepDiff_differenceExp', dataId=dataId)
+   diaSourceTable = butler.get('deepDiff_diaSrc', dataId=dataId)
+
+
+.. _section-ap-pipe-calexp-templates-gen2:
+
+Calexp template mode
+====================
+
+By default, ``ap_pipe`` assumes you would like to use PSF-matched coadd images
+as templates for difference imaging. However, the pipeline also supports
+using calibrated exposures (``calexps``) as templates instead. A configuration file
+``config/calexpTemplates.py`` is included witha ``ap_pipe`` to enable this.
+
+To use ap_pipe in calexp template mode, point to the config file with the
+``--configfile`` (``-C``) flag and additionally specify the ``dataId`` of the template
+with the ``--templateId`` flag, e.g.,
+
+.. code-block:: none
+
+   -C $AP_PIPE_DIR/config/calexpTemplates.py --templateId visit=234567
+
+Be sure to also specify the location of the repo containing the calexp templates
+with the ``--template`` flag if they are not in the main repo.
+A full command looks like
+
+.. prompt:: bash
+
+   ap_pipe.py repo --calib repo/calibs --rerun processed -C $AP_PIPE_DIR/config/calexpTemplates.py -c diaPipe.apdb.isolation_level=READ_UNCOMMITTED -c diaPipe.apdb.db_url="sqlite:///apdb/association.db" --id visit=123456 ccdnum=42 filter=g --template /path/to/calexp/templates --templateId visit=234567
+
```

Comment:
Suggest just deleting this since it's not really supported/preferred

Reply:
Calexp differencing _is_ something people occasionally ask about, e.g. on CLO. I think as long as it works in Gen 2, we might as well document it.

Reply:
That's true, I just haven't ever done it or made sure these directions work. Maybe you could add a quick warning box that this functionality is not actively maintained and may go away in the future?

Reply:
>  This functionality is available in the Gen 2 alert production pipeline, but is not tested as thoroughly as coadd templates. For technical reasons, use of calexp templates will not be supported in the Gen 3 alert production pipeline.